### PR TITLE
fix(codelabs/gce): Update GCE codelab images, links and text.

### PR DESCRIPTION
**Pull request by duftler**

### CODE REVIEW

File: guides/tutorials/codelabs/gce-source-to-prod/index.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ sidebar:
 
 {% include toc %}
 
-In this codelab, you will create a cohesive workflow which takes source code and builds, tests and promotes it to production. You can also watch a videos of this codelab in this [Youtube playlist](https://www.youtube.com/watch?v=N9VnJlKn734&list=PL4yLrwUObNkttE526AAj_ykc5UlIPjz8m&index=1).
+`In this codelab, you will create a cohesive workflow which takes source code and builds, tests and promotes it to production. You can also watch a video of this codelab in this [Youtube playlist](https://www.youtube.com/watch?v=N9VnJlKn734&list=PL4yLrwUObNkttE526AAj_ykc5UlIPjz8m&index=1).
```
Comment (ttomsu) [inline guides/tutorials/codelabs/gce-source-to-prod/index.md]: This fronting backtick is probably not what you want.

File: guides/tutorials/codelabs/gce-source-to-prod/index.md [hunk 2]
```diff
@@ -309,8 +299,9 @@ We deploy the image that was picked, to the prod cluster.
 * Select “Deploy” in the *Type* drop down.
 * In the *Server group* section, click *Add server group*.
 * In the dialog that appears, click *Continue without a template*.
-* In the Basic Settings section, enter “prod” in the *Stack* field.
+* In the Basic Settings section, enter "prod" in the *Stack* field.
```
Comment (ttomsu) [inline guides/tutorials/codelabs/gce-source-to-prod/index.md:302]: Should these be ordered lists instead of unordered?

**Review by ttomsu — APPROVED** [review]

File: guides/tutorials/codelabs/gce-source-to-prod/index.md (hunk 1 above)
Comment (duftler) [reply guides/tutorials/codelabs/gce-source-to-prod/index.md]: yep

File: guides/tutorials/codelabs/gce-source-to-prod/index.md (hunk 2 above)
Comment (duftler) [reply guides/tutorials/codelabs/gce-source-to-prod/index.md:302]: I prefer it unordered, but could be convinced if somebody feels strongly about it.

**Pull request merged**